src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -152,7 +152,10 @@ public static long getCurrentTimeTrackerValue()
   }
 
   /**
-   * Get the historical log data for all tag groups within the next FIFO queue time span.
+   * Get the historical log data for all tag groups within the next FIFO queue time span. The
+   * operations performed in this method use a significant amount of memory and it is recommended
```

Comment:
would consume be a better word than use?

Reply:
changed